crates/ruma-federation-api/src/discovery.rs

```diff
@@ -34,7 +34,7 @@ pub struct OldVerifyKey {
     /// Timestamp when this key expired.
     pub expired_ts: MilliSecondsSinceUnixEpoch,
 
-    /// The Unpadded Base64 encoded key.
+    /// The unpadded base64-encoded key.
     pub key: String,
```

Comment:
Why isn't this bit replaced with Base64?

Reply:
Wasn't intentional, thanks for catching :)